deploy/olm-catalog/qdr-operator/0.1.0/catalog-source.yaml

```diff
@@ -50,48 +51,84 @@ items:
                   version: v1
                 specDescriptors:
                 - description: The role and placement plan for the interconnect deployment
-                  displayName: DeploymentPlan
-                  path: deploymentPlan
+                  displayName: Deployment Plan Image
```

Comment:
The specDescriptor does not seem right. Should it describe just the high level spec elements or should it also expand the paths for each entity? This is currently a mix of the two.

Reply:
for that Spec Descriptor field.  If we put in the high level spec element like " DeploymentPlan" the OLM page will collapse, it seems the OLM can't parse the struct object, and I found no x-descriptors for a custom struct object.  
So I selectively choose certain fields to display, and the path now is like 'path: deploymentPlan.image'.   
I think we don't need to display all of those fields, but we really need to figure out which fields are most important to show to client and come back to make the necessary changes.